src/components/tooltip/Marker.js

```diff
@@ -620,6 +620,15 @@ function getMarkerDetails(
         }
         break;
       }
+      case 'IPC': {
+        tooltipDetails = (
+          <TooltipDetails>
+            <TooltipDetail label="Type">{data.messageType}</TooltipDetail>
+            <TooltipDetail label="Sync">{data.sync.toString()}</TooltipDetail>
```

Comment:
I believe this is crashing in the current deploy previews. Is there an updated one?

Reply:
Fixed! This was because I forgot to update the profile being used by the preview. Here's a new one: https://deploy-preview-2172--perf-html.netlify.com/public/dab7795ca9494375f62e4adfe7826e94633825aa